src/compiler/xquery/node-constructor/node-constructor.xsl

```diff
@@ -116,7 +116,7 @@
       Local templates
    -->
 
-   <xsl:template name="local:avt-or-tvt" as="node()+">
+   <xsl:template name="local:avt-or-tvt" as="element(x:dummy)">
```

Comment:
After removing `/@vt`, using a more descriptive element name than `x:dummy` will improve understanding of the compiled query. Maybe `x:avt-or-tvt` or simply `x:vt`?

Reply:
Good idea. I updated the name via 267021f17a29738316c85c20cf2dcfe5d7392739.